src/utils/rfq_blacklist_utils.ts

```diff
@@ -0,0 +1,47 @@
+import { Connection } from 'typeorm/connection/Connection';
+
+import { BlockedAddressEntity } from '../entities/BlockedAddressEntity';
+
+/**
+ * RfqBlacklistUtils helps manage the RFQ blacklist
+ */
+export class RfqBlacklistUtils {
+    private static _instance: RfqBlacklistUtils;
```

Comment:
is this a singleton pattern?

Reply:
do we need it? can we just initialize the class once and pass by reference?

Reply:
good point, this is unnecessary because all of this is in the API and we full control of the reference. Will remove. I was just blindly copying a pattern from elsewhere yesterday when I got tired